### DISCUSSION TITLE
Sortable Node is disappearing after dragging out of small area

### DISCUSSION THREAD

**Discussion by ashater1** [clauderic/dnd-kit#665] [Q&A] [answered]
Hey, love the library!  One thing that's come up that I've been having trouble with - I'm try to use a basic Sortable component, but dragging it more than a few pixels in one direction or another causes it to drop from the cursor.

A really basic example of this in a sandbox here: https://codesandbox.io/s/recursing-dream-1jfbfw?file=/src/App.js

Any help would be greatly appreciated!
![dndkit-sortable](https://user-images.githubusercontent.com/37031895/159153203-787b09aa-4ed6-4e0e-85e0-d29b98cef447.gif)

**Comment by clauderic** [comment, marked as answer]
This is happening because the default [collision detection algorithm](https://docs.dndkit.com/api-documentation/context-provider/collision-detection-algorithms) for `<DndContext>` is `rectIntersection`. This collision detection algorithm requires both the draggable bounding box and the droppable bounding box to be intersecting to register a collision.

The behaviour you're seeing where the sortable item is returning to its initial position is happening because of this. As soon as the active sortable item is no longer intersecting with a droppable, it returns to its initial position because it isn't colliding with anything, so the `over` property is `null`.

For optimal user experience when building sortable interfaces, we recommend using a more permissive collision detection algorithm such as [`closestCenter`](https://docs.dndkit.com/api-documentation/context-provider/collision-detection-algorithms#closest-center).

https://codesandbox.io/s/festive-field-xk760z?file=/src/App.js

**Reply by ashater1** [reply]
Ah, not sure how I missed that - thanks for the help!
